README.md

```diff
@@ -47,6 +48,22 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag's Configuration
+
+The tag's historical logs are the source for the data pushed to OSIsoft, so settings for your historical logging will affect what data can get sent to OSIsoft. Make sure you enable historical logging in your tag settings. 
+1. On the flexy web page, navigate to Tags -> values
+![Tag configuration step 1](docs/images/step1.PNG)
+2. Via the button on the top left section of the web page, switch the mode from view to setup
+![Tag configuration step 2](docs/images/step2.PNG)
+3. Click on the tag that you wish to configure, and click on edit
+4. On the tag configuration panel, scroll down and enable Historical Logging
+![Tag configuration step 3](docs/images/step3.PNG)
+We suggest you make use of both the available historical logging settings: Logging Deadband and Logging Interval. More information on Logging Deadband is listed below.
+
+#### Logging Deadband
+
+Logging Deadband refers to the amount of change needed before values are logged to the historical logs. If the Logging Deadband is set to 1, the value needs to change by 1 or more before being logged. If you have logging deadband disabled (set to -1), we recommend you set the historical log's logging interval setting to something high if possible. Set the deadband to whatever your lowest unit of interest is.
```

Comment:
we - use of first person

Reply:
change to "it is recommended the historical log's logging interval is set to a high value" or ".. the highest value applicable for the application"